core/src/test/java/io/questdb/griffin/AsOfJoinTest.java

```diff
@@ -292,6 +298,49 @@ public void testExplicitTimestampIsNotNecessaryWhenSpliceJoiningExplicitlyOrdere
         testExplicitTimestampIsNotNecessaryWhenJoining("splice join", null);
     }
 
+    @Test
+    public void testFullLtJoinDoesNotConvertSymbolKeyToString() throws SqlException {
+        compiler.setFullFatJoins(true);
+        compile("create table tab_a (sym_a symbol, ts_a timestamp, s_a string) timestamp(ts_a) partition by DAY");
+        compile("create table tab_b (sym_b symbol, ts_b timestamp, s_B string) timestamp(ts_b) partition by DAY");
+        compile("insert into tab_a values ('ABC', '2022-01-01T00:00:00.000000Z', 'foo')");
+        compile("insert into tab_b values ('ABC', '2021-01-01T00:00:00.000000Z', 'bar')");
```

Comment:
Here matching symbols are inserted first, they have the same id.
Can you change the test so that for example `tab_b` has another row with the symbol value `DCE` first?

Reply:
good idea